.repo-metadata.json

```diff
@@ -1,15 +1,15 @@
 {
-  "name": "bigquery-connection",
+  "name": "bigqueryconnections",
   "name_pretty": "Cloud BigQuery Connections",
   "product_documentation": "https://cloud.google.com",
   "api_description": "is about",
-  "client_documentation": "https://googleapis.dev/java/google-cloud-bigquery-connection/latest/index.html",
+  "client_documentation": "https://googleapis.dev/java/google-cloud-bigqueryconnections/latest/index.html",
   "release_level": "beta",
   "transport": "grpc",
   "language": "java",
-  "repo": "googleapis/java-bigquery-connection",
-  "repo_short": "java-bigquery-connection",
-  "distribution_name": "com.google.cloud:google-cloud-bigquery-connection",
-  "api_id": "bigquery-connection.googleapis.com",
+  "repo": "googleapis/java-bigqueryconnections",
+  "repo_short": "java-bigqueryconnections",
+  "distribution_name": "com.google.cloud:google-cloud-bigqueryconnections",
+  "api_id": "bigqueryconnections.googleapis.com",
```

Comment:
this should match whatever the api's endpoint is

Reply:
thanks. upon checking proto, it should be bigqueryconnection - updating this